todoman/model.py

```diff
@@ -322,7 +322,10 @@ def save(self, list_=None):
         list_ = list_ or self.list
         path = os.path.join(list_.path, self.filename)
         assert path.startswith(list_.path)
-
+        sequence = self.todo.get('SEQUENCE')
+        if sequence is None:
+            sequence = 0
```

Comment:
These three lines can actually all be simplified to `sequence = self.todo.get('SEQUENCE', 0)`. What it does is return the sequence, of if it's `None`, return the second value (which is `0`).